aten/src/ATen/DLConvertor.cpp

```diff
@@ -71,6 +71,9 @@ DLDataType getDLDataType(const Tensor& t) {
     case ScalarType::Float8_e8m0fnu:
       TORCH_CHECK(false, "float8 types are not supported by dlpack");
       break;
+    case ScalarType::Float4_e2m1fn_x2:
+      TORCH_CHECK(false, "float4 types are not supported by dlpack");
```

Comment:
float4 are added in https://github.com/dmlc/dlpack/releases/tag/v1.1 Is it possible to add it?

Reply:
@vkuzo will be away for a few weeks, do you want to submit PR @justinchuby?